src/app/interactive.clj

```diff
@@ -33,6 +33,17 @@
 (def no-classes-loaded-error
   "No classes loaded. Use <load> first")
 
+(def help-message
```

Comment:
maybe here we could join from clojure.string:

`(join "\n" ["h/help   print this help" "q/quit/exit....", etc])`

So we do not have to repeat the `\n`